client/src/main.ts

```diff
@@ -1,15 +1,16 @@
 import Vue from 'vue'
 import App from './App.vue'
-import router from './router'
+import { getVueRouter } from '@/router'
 import store from './store'
 import { getConfig } from '@/plugins/getConfig'
-import vuetify from './plugins/vuetify'
+import Vuetify from "vuetify/lib"
 import KeyCloakService from 'sbc-common-components/src/services/keycloak.services'
 
-import 'quill/dist/quill.core.css'
+import '@mdi/font/css/materialdesignicons.min.css' // ensure you are using css-loader
 import 'material-design-icons-iconfont/dist/material-design-icons.css'
```

Comment:
Is this obsolete?

Reply:
No, not quite. A lot of the dropdown arrows and form icons are utilizing this. I didn't want to comb through everything icon related in this ticket because it extended a tad bit outta this tickets scope.

Reply:
OK. If you haven't already, please create a tech debt ticket to remove this properly.